packages/core/test/merkle-distributor/MerkleDistributor.js

```diff
@@ -128,8 +128,8 @@ contract("MerkleDistributor.js", function(accounts) {
       assert.equal(windowState.rewardToken, rewardToken.address);
       assert.equal(windowState.ipfsHash, sampleIpfsHash);
 
-      // Check that latest created index has incremented.
-      assert.equal((await merkleDistributor.lastCreatedIndex()).toString(), (windowIndex + 1).toString());
+      // Check that next created index has incremented.
```

Comment:
please also update this script:
https://github.com/UMAprotocol/protocol/blob/69b87e98283b6970375358010d906f55fa29dd9b/packages/merkle-distributor/scripts/2_PublishClaimsForWindow.ts#L81